base/client.jl

```diff
@@ -117,6 +123,7 @@ function eval_user_input(errio, @nospecialize(ast), show_value::Bool)
                 print(color_normal)
             end
             if lasterr !== nothing
+                ccall(:jl_set_global, Cvoid, (Any, Any, Any), Main, :errs, ExceptionInfo(lasterr))
```

Comment:
What do you think about `lasterr`?

Reply:
I had feedback that `lasterr()` was too much typing, so I went to `err`; then @vtjnash suggested `errs`, which in theory it could contain multiple.  I suppose that might depend on how frequently someone would need to type it, or whether something like Cthulhu is intended to be the option of choice for debugging the detailed traces? I'm open to whatever.

Reply:
As `ans` is not called `lastans`, `err` or `errs` seems fine to me.

Reply:
In that case I would prefer `err`. Even though `errs` is perhaps technically more correct, it feels strange.